start-from-zero/Javascript/src/data/data.js

```diff
@@ -86,7 +86,7 @@ function FieldsetData(){
                         span: 'Classes',
                         name: 'classes',
                         optionValue: ['0', '1', '2', '3'],
-                        dropdown: ['Javascript', 'Java', 'Python', 'Haskell'],
+                        dropdown: ['xbox', 'Java', 'Python', 'Haskell'],
```

Comment:
can't wait to have my first xbox interview

Reply:
:))